srcpkgs/json-c/template

```diff
@@ -1,21 +1,30 @@
-# Template build file for 'json-c'.
+# Template file for 'json-c'
 pkgname=json-c
 version=0.13.1
-revision=1
+revision=2
 build_style=gnu-configure
+configure_args="$(vopt_enable threading) $(vopt_enable rdrand)"
 hostmakedepends="automake libtool"
+checkdepends="valgrind"
+make_check_args="--keep-going"
 short_desc="A JSON implementation in C"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="MIT"
-homepage="http://oss.metaparadigm.com/$pkgname"
-distfiles="https://s3.amazonaws.com/json-c_releases/releases/$pkgname-$version.tar.gz"
+homepage="https://${pkgname}.github.io/${pkgname}/"
+distfiles="https://s3.amazonaws.com/${pkgname}_releases/releases/${pkgname}-${version}.tar.gz"
 checksum=b87e608d4d3f7bfdd36ef78d56d53c74e66ab278d318b71e6002a369d36f4873
 
+build_options="threading rdrand"
+build_options_default="threading rdrand"
```

Comment:
if rdrand is considered to be dangerous on non x86_64|i686 platforms you can guard them under a case/esac structure